### PR TITLE
add start button fcitx changed

### PR DESCRIPTION
**Pull request by chenshijie-uos**
i just use cmd  `clang-format -i ./src/lib/configwidgetslib/erroroverlay.cpp -style=LLVM`.

seem different clang-format code.

### CODE REVIEW

**Comment by wengxt** [comment]
We have .clang-format , no need to use -style . Just make sure you run it under source directory

**Comment by chenshijie-uos** [comment]
ok ，i use right clang-format code style

File: src/lib/configwidgetslib/erroroverlay.ui [hunk 1]
```diff
@@ -211,6 +211,33 @@
...
+    <widget class="QLabel" name="runfcitxLabel">
+     <property name="text">
+      <string/>
+     </property>
+    </widget>
+   </item>
+   <item>
+    <widget class="QPushButton" name="runfcitxButton">
+     <property name="text">
+      <string>run Fcitx5</string>
```
Comment (wengxt) [inline src/lib/configwidgetslib/erroroverlay.ui:224]: "Run Fcitx 5"

File: src/lib/configwidgetslib/erroroverlay.ui [hunk 2]
```diff
@@ -211,6 +211,33 @@
...
    </item>
+   <item>
+    <widget class="QLabel" name="runfcitxLabel">
+     <property name="text">
+      <string/>
+     </property>
+    </widget>
+   </item>
+   <item>
+    <widget class="QPushButton" name="runfcitxButton">
```
Comment (wengxt) [inline src/lib/configwidgetslib/erroroverlay.ui:222]: runFcitxButton

Comment (wengxt) [reply src/lib/configwidgetslib/erroroverlay.ui:222]: Also can you add an icon to the button? You may use system-run as icon name.

**Review by wengxt — CHANGES_REQUESTED** [review]

File: src/lib/configwidgetslib/erroroverlay.ui (hunk 2 above)
Comment (chenshijie-uos) [reply src/lib/configwidgetslib/erroroverlay.ui]: ok

**Comment by chenshijie-uos** [comment]
Is there anything else that needs to be changed?

**Comment by wengxt** [comment]
You can take a look at c7501b4 , for button icon can be set in UI file no need to set in code.

**Pull request merged**